changelog/541.feature.rst

```diff
@@ -0,0 +1,3 @@
+Add basic arithmetic methods between `~ndcube.NDCube` objects and broadcastable arrays
+and `~astropy.unit.Quantity` objects. Operations between two `~ndcube.NDCube` objects
```

Comment:
And scalars, right?

Reply:
I was grouping scalars with "broadcastable arrays" but yes I can note that explicitly.